index.js

```diff
@@ -73,7 +73,23 @@ function attachToBody (options, req, reply, next) {
       mp.destroy(new Error(`${key} is not allowed as field name`))
       return
     }
-    if (body[key] === undefined) {
+
+    const tokens = (
```

Comment:
attachToBody is called only in the preValidation hook in line 277. We could actually add above the addHook a line to normalize the value. Then we dont need to check here for multiple values.

Reply:
Didn't get it. Could you explain it in more details?